server/database/__init__.py

```diff
@@ -0,0 +1,32 @@
+# Import flask and template operators
+from flask import Flask, render_template
+
+# Import SQLAlchemy
+from flask.ext.sqlalchemy import SQLAlchemy
+
+# Define the WSGI application object
+app = Flask(__name__)
```

Comment:
creating another flask server wont link this database to our existing flask server. we should only have 1 server and link them together